packages/cli/test/commands/init.test.js

```diff
@@ -9,9 +9,9 @@ describe('init command', function() {
   itShouldParse(
     'should call init script with name, version and dependencies',
     'init',
-    'zos init MyApp 0.2.0 --force --no-install --link mock-stdlib@1.1.0,mock-stdlib2@1.2.0',
+    'zos init MyApp 0.2.0 --force --no-install --link mock-stdlib@1.1.0,mock-stdlib-2@1.2.0',
```

Comment:
Good catch! `mock-stdlib2` doesn't even exist. Tests were passing though, as this test is only asserting that the parameters sent through the `cli` are the same as the ones received by the scripts.